README.md

```diff
@@ -44,7 +44,7 @@ For other issues, please use the following repos:
 ## Useful Links
 
 * [.NET Core source index](https://source.dot.net) / [.NET Framework source index](https://referencesource.microsoft.com)
-* [API Reference docs](https://docs.microsoft.com/dotnet/api/?view=netcore-3.0)
+* [API Reference docs](https://docs.microsoft.com/dotnet/api/?view=netcore-3.1)
```

Comment:
Is there any way to link to the latest version, whatever it may be, rather than hardcoding a specific one? e.g. so that this link will automatically go to the docs for .NET 5 when they're available?